app/ashlar/models.py

```diff
@@ -25,8 +25,9 @@ class Meta(object):
 
 
 class SchemaModel(AshlarModel):
-    version = models.IntegerField()
+    version = models.PositiveIntegerField()
     schema = JsonBField()
+    next_version = models.OneToOneField('self', related_name='previous_version', null=True, editable=False)
```

Comment:
Good catch, I think I may have quit my editor without giving the linter a chance to work.